wiki/FitNesseRoot/HsacExamples/SlimTests/BrowserTests.wiki

```diff
@@ -54,14 +54,16 @@ The [[heuristic to find places][>FindingByHeuristics]] tries a number of transla
 |page title                                                                |retrieves the current page's title (for use with 'check' verb)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
 |is visible <place>                                                        |whether specified place is visible in browser's window (used in combination with 'ensure' and 'reject')                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
 |is visible on page <place>                                                |whether specified place is visible on the page in the browser's window, scrolling may be required to actually show it (used in combination with 'ensure' and 'reject')                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+|not visible <place> (in <container>)                                      |whether specified place is not visible in browser's window (used in combination with 'ensure' and 'reject'). Will retry for <seconds before timeout> seconds if place is visible.                                                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
Can we merge these descriptions with 'is visible' and 'is visible on page'? It seems a bit too much to have 4 rows for these methods

Reply:
Merged. See if you think it's clearly dcumented like this